lib/RegisterAll.cpp

```diff
@@ -36,6 +37,13 @@ void mlir::tt::registerAllDialects(mlir::DialectRegistry &registry) {
 #if TTMLIR_ENABLE_STABLEHLO
   mlir::stablehlo::registerAllDialects(registry);
 #endif
+  arith::registerBufferizableOpInterfaceExternalModels(registry);
```

Comment:
Can you comment why this is needed.

Reply:
Note that these shouldn't have been in this PR at all, mis-cherry-pick issue from me.  However, I just merged PR where they should be, so adding comment here anyway.